http2-client-grpc/src/Network/GRPC/Client.hs

```diff
@@ -307,11 +308,11 @@ sendSingleMessage rpc msg encoding flagMod conn connectionFlowControl stream str
             got       <- _withdrawCredit connectionFlowControl gotStream
             liftIO $ _receiveCredit streamFlowControl (gotStream - got)
             if got == wanted
-            then
-                sendData conn stream flagMod dat
-            else do
-                sendData conn stream id (ByteString.take got dat)
-                goUpload (ByteString.drop got dat)
+                then
+                    sendData conn stream flagMod dat
+                else do
+                    sendData conn stream id (ByteString.take got dat)
+                    goUpload (ByteString.drop got dat)
```

Comment:
I don't get why this was changed. It's just a style change, no better, no worse, hence it should not be done.

I'm guessing the previous style is consistent in the project.

Reply:
The compiler does not agree with you unfortunately 😉

Reply:
I am rechecking...

Reply:
Interestingly, it does not complain now, reverting...